UnityProject/Assets/Scripts/LevelCreatorScript.cs

```diff
@@ -236,6 +236,8 @@ public class TileInstance {
     		CreateTileIfNeeded(tile_x+i);
     	}
 
+        if(PlayerPrefs.GetInt("shadowed_lights", 1) == 0)
+            return;
```

Comment:
Probably better and clearer to do a if block here, executing the related code inside that, rather than an early return. Early returns aren't as clear about their flow and code added in the future at the end of the function may stop functioning without the programmer realizing why.